uxarray/grid.py

```diff
@@ -73,9 +73,20 @@ def __init__(self, dataset, **kwargs):
 
         # check if initializing from verts:
         if isinstance(dataset, (list, tuple, np.ndarray)):
-            self.vertices = dataset
-            self.__from_vert__()
-            self.source_grid = "From vertices"
+            dataset = np.asarray(dataset)
+            if dataset.ndim == 3:
+                # This Grid contains multiple faces
+                self.__from_vert__(dataset)
+                self.source_grid = "From vertices"
+            elif dataset.ndim == 2:
+                dataset = np.array([dataset])
+                self.__from_vert__(dataset)
+                self.source_grid = "From vertices"
+            else:
+                raise RuntimeError(
+                    "Input vertices don't have correct dimensions (Expected dimension will be 3: [nMesh2_face, "
+                    "nMesh2_node, Two/Three]). Or dimesion of 2 when only one face is passed"
+                )
```

Comment:
Any way that you could split up this exception to catch the "Or dimesion of 2 when only one face is passed" more directly?

Reply:
"Or dimesion of 2 when only one face is passed" is not an exception, it's what's allowed

Basically this exception error is just telling users we can only accept the dimension as 2 or 3. And the `ndim==2` case only applies to when a single face is passed

But feel free to rephrase this sentence to make the message more clear